config.py

```diff
@@ -2201,6 +2299,39 @@ def check_baseline(
             )
 
 
+if decoding_n_splits < 2:
+    raise ValueError('decoding_n_splits should be at least 2.')
```

Comment:
can you check that setting decoding_n_splits=1 in a custom config raises the exception?

Reply:
Confirmed that it works